abi.md

```diff
@@ -0,0 +1,146 @@
+---
+author:
+- The FIRRTL Specification Contributors
+title: Specification for the FIRRTL ABI
+revisionHistoryAbi: true
+---
+
```

Comment:
@seldridge does something need to be added here to make the revision history show up? Or does that happen automatically due to what you did in the previous PR?

Reply:
Nothing else needs to be added. I landed the support earlier. This specific config option was added in https://github.com/chipsalliance/firrtl-spec/commit/9bb1272145484afcd698015e2b6de648123629ee.

This is mega-jank (due to limitations of using a single LaTeX template, what pandoc lets you do, and my own skill), but there are two configuration options you can define up front:

```yaml
revisionHistoryAbi:
revisionHistorySpec:
```

Each of these, if defined, controls, respectively, whether the `abi` or `spec` parts of the `revision-history.yaml` are included in that Revision History section of a document. This clearly doesn't scale well to multiple documents and it would be nice to customize the LaTeX template based on the value associated with a key as opposed to the existence of a key (but pandoc doesn't let you do this as far as I know). It's good enough for today, though.

Reply:
Here is what this looks like in the built `.pdf`s:

<img width="1195" alt="Screenshot 2023-05-03 at 15 24 43" src="https://user-images.githubusercontent.com/1018530/236023590-fedf6f8e-cd71-4a8f-acd6-2c60082bd259.png">
<img width="1185" alt="Screenshot 2023-05-03 at 15 25 21" src="https://user-images.githubusercontent.com/1018530/236023593-3d4439f1-4d2c-4f4e-a1cb-9542333a0d01.png">

Reply:
Looks great!